app/views/vm_common/_evacuate.html.haml

```diff
@@ -0,0 +1,53 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "vmCloudEvacuateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Evacuate Host')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto-select Host?')
+      .col-md-8
+        %input{:type         => "checkbox",
+               :name         => "auto_select_host",
+               'ng-model'    => "vmCloudModel.auto_select_host",
+               :miqrequired  => true,
+               :checkchange  => true,
+               'ng-disabled' => "hosts.length == 0"}
```

Comment:
Hm, I didn't realize that ng-disabling that input results in it not being sent. That's a little aggravating.